api/HTMLSelectElement.json

```diff
@@ -0,0 +1,1306 @@
+{
+  "api": {
+    "HTMLSelectElement": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLSelectElement",
+        "support": {
+          "webview_android": {
+            "version_added": "1"
+          },
+          "chrome": {
+            "version_added": "1"
+          },
+          "chrome_android": {
+            "version_added": "1"
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "2",
+            "notes": "Historically, Firefox has allowed keyboard and mouse events to bubble up from the <code>&lt;option&gt;</code> element to the parent <a href='https://developer.mozilla.org/docs/Web/HTML/Element/select' title='The HTML <select> element represents a control that provides a menu of options:'><code>&lt;select&gt;</code></a> element. This doesn't happen in Chrome, however, although this behavior is inconsistent across many browsers. For better Web compatibility (and for technical reasons), when Firefox is in multi-process mode and the <code>&lt;select&gt;</code> element is displayed as a drop-down list. The behavior is unchanged if the <code>&lt;select&gt;</code> is presented inline and it has either the <code>multiple</code> attribute defined or a <code>size</code> attribute set to more than 1. Rather than watching <code>&lt;option&gt;</code> elements for events, you should watch for <a href='https://developer.mozilla.org/docs/Web/Events/change' title='The change event'>change</a> events on <a href='https://developer.mozilla.org/docs/Web/HTML/Element/select' title='The HTML <select> element represents a control that provides a menu of options:'><code>&lt;select&gt;</code></a>. See <a href='https://bugzil.la/1090602' title='[e10s] <option> events do not bubble up through parent <select>' >bug&nbsp;1090602</a> for details."
```

Comment:
For the record, rather than go back and forth on improving this, I'm putting rewriting this giant note on my to do list 😃